engines/e_capi.c

```diff
@@ -462,7 +462,11 @@ static int capi_init(ENGINE *e)
 
 # ifdef OPENSSL_CAPIENG_DIALOG
     {
-        HMODULE cryptui = LoadLibrary(TEXT("CRYPTUI.DLL"));
+#if defined(OPENSSL_SYS_WIN_CORE)
+        HMODULE cryptui = LoadLibraryEx(TEXT("CRYPTUI.DLL"), NULL, 0);
```

Comment:
As already said, there is LoadLibrary in onecore.lib, so why is it needed? However, if it turns out that it is needed, it would be more appropriate to just switch to LoadLibraryEx without any #ifdefs. It was around for long enough not to care about its availability.